server/src/test/java/org/kaazing/gateway/server/context/resolve/DefaultClusterContextTest.java

```diff
@@ -281,8 +282,8 @@ public void shouldSeeBalancerStateAfterClusterMembersLeave() throws Exception {
                     connects4,
                     null);
 
-            startClusterContext(clusterContext1);
-            addToClusterState(clusterContext1.getCollectionsFactory(), clusterContext1.getLocalMember(), 1);
+            Map<ClusterContext, String> clusterContextToServiceUrlMap = new HashMap<>();
```

Comment:
This holds the expected balanced services for each node. Should be renamed accordingly.